scripts/install_test_dependencies.sh

```diff
@@ -15,8 +15,19 @@
 
 set -ex
 
-sudo apt-get update
-sudo apt-get install -y php5-cli
+if [ "${CIRCLECI}" != "true" ]; then
+    sudo apt-get update
+    sudo apt-get install -y php5-cli
+    # Install gcloud
```

Comment:
Looks like this code path only execute if we aren't on CircleCI. What is this for?